src/modules/LanItems.rb

```diff
@@ -2382,6 +2406,34 @@ def self.publish_variable(name, type)
 
   private
 
+    # Returns a formated string with the interfaces that are part of a bridge
+    # or of a bond interface.
+    #
+    # @param [String] ifcfg_type
+    # @param [String] ifcfg_name
+    # @return [String] formated string with the interface type and the interfaces enslaved
+    def slaves_desc(ifcfg_type, ifcfg_name)
+      if ifcfg_type == "bond"
+        slaves = GetBondSlaves(ifcfg_name)
+        desc = _("Bonding slaves")
+      else
+        slaves = bridge_slaves(ifcfg_name)
+        desc = _("Bridge Ports")
+      end
+
+      format("%s: %s", desc, slaves.join(" "))
+    end
+
+    # Check if the given interface is enslaved in a bond or in a bridge
+    #
+    # @return [Boolean] true if enslaved
+    def enslaved?(ifcfg_name)
+      bond_index = BuildBondIndex()
+      return true if bond_index[ifcfg_name] || bridge_index[ifcfg_name]
```

Comment:
well, my idea behind this is that one day there will be a generic ```iface_type``` class with ```bridge_type``` and ```bond_type``` successors. If you rewrite it into two, we can copy paste methods as they are into these classes with almost no change.

If it makes sense to you split it. If you think that I'm overengineering things keep it as it is.